crashlog/crashutils.c

```diff
@@ -161,8 +161,33 @@ unsigned long long get_uptime(int refresh, int *error)
     return time_ns;
 }
 
+// Find system last kmsg from dropbox
+static int find_system_last_kmsg(char *source) {
```

Comment:
How do we make sure copy length not exceed input buffer length?

Reply:
Resolved, thanks.